tests/unit/test_discovery_client.py

```diff
@@ -16,8 +16,19 @@
     "ni.measurementlink.measurement.v2.MeasurementService",
 ]
 
+_PROVIDED_ANNOTATIONS = {
```

Comment:
The annotations map in the request object is string to string.  All of the values needs to be changes to embedded JSON strings.